compass/landice/tests/humboldt/mesh.py

```diff
@@ -0,0 +1,361 @@
+                database='')
+        self.add_input_file(filename='Humboldt.geojson',
+                            package='compass.landice.tests.humboldt',
+                            target='Humboldt.geojson',
+                            database=None)
+        self.add_input_file(filename='greenland_8km_2020_04_20.epsg3413.nc',
+                            target='greenland_8km_2020_04_20.epsg3413.nc',
+                            database='')
+
+    # no setup() method is needed
+
+    def run(self):
+        """
+        Run this step of the test case
+        """
+        logger = self.logger
+        config = self.config
+        section = config['humboldt']
+
+        logger.info('calling build_cell_wdith')
+        cell_width, x1, y1, geom_points, geom_edges = self.build_cell_width()
+        logger.info('calling build_planar_mesh')
+        build_planar_mesh(cell_width, x1, y1, geom_points,
+                          geom_edges, logger=logger)
+        dsMesh = xarray.open_dataset('base_mesh.nc')
+        logger.info('culling mesh')
+        dsMesh = cull(dsMesh, logger=logger)
+        logger.info('converting to MPAS mesh')
+        dsMesh = convert(dsMesh, logger=logger)
+        logger.info('writing grid_converted.nc')
+        write_netcdf(dsMesh, 'grid_converted.nc')
+        # If no number of levels specified in config file, use 10
+        levels = section.get('levels')
+        logger.info('calling create_landice_grid_from_generic_MPAS_grid.py')
+        args = ['create_landice_grid_from_generic_MPAS_grid.py',
+                '-i', 'grid_converted.nc',
+                '-o', 'gis_1km_preCull.nc',
+                '-l', levels, '-v', 'glimmer']
+        check_call(args, logger=logger)
+
+        # This step uses a subset of the whole Greenland dataset trimmed to
+        # the region around Humboldt Glacier, to speed up interpolation.
+        # This could also be replaced with the full Greenland Ice Sheet
+        # dataset.
+        logger.info('calling interpolate_to_mpasli_grid.py')
+        args = ['interpolate_to_mpasli_grid.py', '-s',
+                'humboldt_1km_2020_04_20.epsg3413.icesheetonly.nc', '-d',
+                'gis_1km_preCull.nc', '-m', 'b', '-t']
+
+        check_call(args, logger=logger)
+
+        # This step is only necessary if you wish to cull a certain
+        # distance from the ice margin, within the bounds defined by
+        # the GeoJSON file.
+        cullDistance = section.get('cullDistance')
+        if float(cullDistance) > 0.:
+           logger.info('calling define_cullMask.py')
+           args = ['define_cullMask.py', '-f',
+                   'gis_1km_preCull.nc', '-m'
+                   'distance', '-d', cullDistance]
+
+           check_call(args, logger=logger)
+        else:
+           logger.info('cullDistance <= 0 in config file. ' \
+                       'Will not cull by distance to margin. \n')
```

Comment:
```suggestion
        if float(cullDistance) > 0.:
            logger.info('calling define_cullMask.py')
            args = ['define_cullMask.py', '-f',
                    'gis_1km_preCull.nc', '-m'
                    'distance', '-d', cullDistance]

            check_call(args, logger=logger)
        else:
            logger.info('cullDistance <= 0 in config file. ' \
                        'Will not cull by distance to margin. \n')
```
This needs the proper level of indentation.